frontend/src/components/messstelle/gesamtauswertung/stepper/OrtStepContent.vue

```diff
@@ -192,9 +211,10 @@ const messstelleHint = computed(() => {
 
 const disableMessstelle = computed(() => {
   return (
-    auswertungOptions.value.mqIds.length !== 0 &&
+    auswertungOptions.value.messstelleAuswertungIds.length > 0 &&
+    auswertungOptions.value.messstelleAuswertungIds[0].mqIds.length !== 0 &&
```

Comment:
Weshalb wird hier `!== 0` und in Zeile 217 `> 0` verwendet? Es handelt sich um die selbe Prüfung.